portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -74,11 +86,16 @@ private int getNumberComments(HttpServletRequest request) {
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String userComment = request.getParameter("comment");
+    String userName = request.getParameter("name");
     long timestamp = System.currentTimeMillis();
 
+    String id = userName + userComment;
```

Comment:
This will not provide unique IDs for your comments. If someone posts the same thing multiple times, you will get conflicting IDs.

One pattern that's often used is to start the IDs at 0 and then increment it by 1 for every new entry. The important part is to have some way of reliably generating new, previously unused IDs.